### DISCUSSION TITLE
Can OC support LinkedIn integration?

### DISCUSSION THREAD

**Discussion by orchardc** [OrchardCMS/OrchardCore#12530] [Q&A] [answered]
What I need is, users can register and login via their linkedin account. Is it supported in OC?

**Comment by hishamco** [comment, marked as answer]
It's not supported OOB but you implement your custom module for doing that, please check other authentication providers for more info

**Comment by ns8482e** [comment]
OC supports OpenId client https://docs.orchardcore.net/en/dev/docs/reference/modules/OpenId/#oidc-client

Using that you can connect to any external identity provider that supports OpenId
